traitsui/testing/tester/qt4/_traitsui/range_editor.py

```diff
@@ -16,12 +16,70 @@
     SimpleSliderEditor,
 )
 
-from traitsui.testing.tester import locator
-from traitsui.testing.tester.qt4 import _registry_helper
-from traitsui.testing.tester.qt4.common_ui_targets import (
-    LocatedSlider,
-    LocatedTextbox
-)
+from traitsui.testing.tester import command, locator
+from traitsui.testing.tester.qt4 import _interaction_helpers, _registry_helper
```

Comment:
An observation entirely out-of-scope for this PR: strange mismatch between "helpers" and "helper" there. Since they are now obviously private, we should not need to worry too much about breaking external code if we want to rename them after release. The names still provide good enough hints to maintain encapsulation and to guide code discovery. So the little difference here bother those who suffer from OCD (who cares about them 😂 )

Reply:
I agree, I also dislike the mismatch.  We can make a fix in a separate PR